reference/rebac_admin_backend.rst

```diff
@@ -23,13 +23,14 @@ JAAS Implementation
 
 JAAS implements a subset of the operations described in the OpenAPI spec. 
 
- ====== ================ =========================================================================
- Status Entities         Notes
- ====== ================ =========================================================================
+ ====== ================== =========================================================================
+ Status Entities           Notes
+ ====== ================== =========================================================================
    ✅   ``entitlements``     
    ✅   ``capabilities`` 
    ✅   ``groups``     
    ✅   ``resources``     
-   🟡   ``identities``    no support for creation, update and deletion.
-   ❌   ``roles``         no support for roles.
- ====== ================ =========================================================================
+   🟡   ``identities``     no support for creation, update and deletion.
+   ❌   ``roles``          no support for roles.
+   ❌   ``authentication`` no support for authentication providers.
```

Comment:
What are authentication providers in the rebac world btw?

Reply:
![image](https://github.com/user-attachments/assets/89c114da-752a-4f58-b92e-c34bd2c735d4)

the swagger is not public yet, i paste here the screenshot to provide more context

Reply:
Maybe, instead of `authentication`, we could say `no support for identity provider management`

Reply:
Yeah, I agree with @alesstimec's suggestion.

Reply:
@kian99 It's more about the supported IdP (those that identities can be retrieved from and added to). And I agree that this is a bit unrelated to ReBAC stuff, but that's already part of the OpenAPI spec.